website/client/components/settings/subscription.vue

```diff
@@ -237,6 +237,10 @@ export default {
         amount: this.numberOfMysticHourglasses,
       };
     },
+    dateTerminated () {
+      if (!this.user.preferences || !this.user.preferences.dateFormat) return moment(this.user.purchased.plan.dateTerminated);
```

Comment:
this should return a string not a moment object, calling `.format()` should be enough